samples/ios/Continote/Podfile.lock

```diff
@@ -5,9 +5,9 @@ PODS:
   - Bolts/AppLinks (1.8.4):
     - Bolts/Tasks
   - Bolts/Tasks (1.8.4)
-  - FBSDKCoreKit (4.23.0):
+  - FBSDKCoreKit (4.24.0):
```

Comment:
I had some discussion with @davidair about this when just starting the iOS sample code.

It seems like the consensus is to include this Podfile.lock file in the repo (i.e. _not_ put it in .gitignore).

For example, see this thread: https://github.com/flutter/flutter/issues/9299

However, given the nature of the file, I completely understand your initial concern. Any *.lock file in a repo gives me pause, too.